apps/server/test/lexical/server/code_intelligence/completion/translations/macro_test.exs

```diff
@@ -118,7 +118,58 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.MacroTest do
       assert completion.insert_text_format == :snippet
 
       assert completion.insert_text == """
-             defmodule ${1:module name} do
+             defmodule ${1:File} do
+               $0
+             end
+             """
+    end
+
+    test "defmodule for test paths", %{project: project} do
+      assert {:ok, completion} =
+               project
+               |> complete("defmodule|", path: "test/foo/project/my_test.exs")
+               |> fetch_completion("defmodule ")
+
+      assert completion.detail
```

Comment:
I found there is no detail here, Just like the struct completions, if we put the `module` name to the detail. that would be helpful.

Reply:
Do you need the detail? The snippet shows the suggested module name, and allows you to change it by typing. Also, nested module names can be quite long, which won't look good in the completion ui

Reply:
I see, I'm fine without `detail`